__tests__/api.test.js

```diff
@@ -104,3 +104,59 @@ describe('GET /api/users', () => {
         })
     })
 })
+
+describe('PATCH /api/reviews/:review_id', () => {
+    test('200: responds with the correctly updated review object when incrementing the review votes', () => {
+        const testInc = {inc_votes: 5};
+        const testReviewID = 10;
+        return request(app)
+        .patch(`/api/reviews/${testReviewID}`)
+        .send(testInc)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.review).toEqual(expect.objectContaining({
+                review_id: 10,
+                title: expect.any(String),
+                review_body: expect.any(String),
+                designer: expect.any(String),
+                review_img_url: expect.any(String),
+                votes: 15,
+                category: expect.any(String),
+                owner: expect.any(String),
+                created_at: expect.any(String)
+            }))
+        })
+    })
+    test('200: responds with the correctly updated review object when decrementing the review votes', () => {
+        const testDec = {inc_votes: -10};
+        const testReviewID = 9;
+        return request(app)
+        .patch(`/api/reviews/${testReviewID}`)
+        .send(testDec)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.review).toEqual(expect.objectContaining({
+                review_id: 9,
+                title: expect.any(String),
+                review_body: expect.any(String),
+                designer: expect.any(String),
+                review_img_url: expect.any(String),
+                votes: 0,
+                category: expect.any(String),
+                owner: expect.any(String),
+                created_at: expect.any(String)
+            }))
+        })
+    })
+    test('400: responds with bad request when invalid increment is provided', () => {
+        const testInc = {inc_votes: 'not_a_number'};
+        const testReviewID = 10;
+        return request(app)
+        .patch(`/api/reviews/${testReviewID}`)
+        .send(testInc)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    })
```

Comment:
You should also include a tests for:
- if the review isn't found
- if it's an invalid review id
- if the inc_vote key is missing from the request